packages/cli/test/acceptance/app-run.acceptance.js

```diff
@@ -15,15 +17,34 @@ describe('app-generator (SLOW)', function() {
   const generator = path.join(__dirname, '../../generators/app');
   const rootDir = path.join(__dirname, '../../../..');
   const sandbox = path.join(rootDir, 'sandbox/sandbox-app');
+  const sandboxDefaultVal = path.join(rootDir, 'sandbox/sandbox-default-value');
   const cwd = process.cwd();
   const appName = '@loopback/sandbox-app';
+
+  /** NOTICE: if this test runs outside of the current user's $HOME directory,
+  then tildifySandbox will be identical to sandbox. Means the tests will pass
+  without testing the ~ support. This could come up on CI systems where
+  tests are run inside /tmp, for example. */
+  // For testing out the untildified path:
+  // use a path that contains ~ as the outdir of the app
+  const tildifySandbox = tildify(sandbox);
   const props = {
     name: appName,
     description: 'My sandbox app for LoopBack 4',
-    outdir: sandbox,
+    outdir: tildifySandbox,
```

Comment:
I see that you are modifying an acceptance test suite that's primarily testing how a new app is scaffolded, whether dependencies can be installed, etc. I feel these tests should not deal with home-directory specific stuff and should be preserved with no changes.

Instead, you should write a new test that will focus only on dealing with home directories. This test can be much simpler and should go to https://github.com/strongloop/loopback-next/blob/spike%2Fresolve-included-models/packages/cli/test/integration/generators/app.integration.js

Ideally, the new test should also modify `process.env.HOME` to ensure we are testing tildify/untildify behavior on CI too, see https://github.com/strongloop/loopback-next/pull/3252#discussion_r300438024

Sorry for discovering this late.